src/__mocks__/mock_puzzle.ts

```diff
@@ -0,0 +1,25 @@
+import { Puzzle } from '@prisma/client';
+
+const puzzle = {
+  id: 1,
+  name: 'fake1',
+  createdAt: new Date(Date.now()),
+  updatedAt: new Date(Date.now()),
+  difficulty: 'EASY',
+  content: [],
+  imageUrl: null,
+  exampleContent: [],
+  exampleImageUrl: null,
+  isGenerated: true,
+  question: 'What is 1+1?',
+  inputType: 'TEXT',
+  published: false,
+  variables: {
+    answer: '2'
+  },
+  puzzleType: {},
+  puzzleTypeId: 1,
+  puzzleInstances: []
+} as Puzzle;
```

Comment:
This still contains a lot of constants. Since my puzzles have been up, you can try to integrate that to your code

Reply:
I didn't delete this as an alternative for mockPuzzle in case "variables" or the answer is empty.

Reply:
okay I deleted this